src/Code/CodeBlock.tsx

```diff
@@ -1,5 +1,5 @@
 import clsx from 'clsx';
-import { ComponentPropsWithoutRef, ForwardedRef, forwardRef, ReactElement } from 'react';
+import React, { ComponentPropsWithoutRef, ForwardedRef, forwardRef, ReactElement } from 'react';
```

Comment:
Why did you import React here?

Reply:
It is to fix this eslint error:  ` 40:5   error  'React' must be in scope when using JSX  react/react-in-jsx-scope`.
The `React` import is not needed in next.js apps but in other react apps it is sometimes required for it to work.

Reply:
Oo gotcha makes sense